src/Language/PureScript/Linter/Exhaustive.hs

```diff
@@ -242,8 +242,6 @@ checkExhaustive env mn cas = makeResult . first nub $ foldl' step ([initial], (p
 
   initial :: [Binder]
   initial = initialize numArgs
```

Comment:
Might as well get rid of this now, and `numArgs` is in scope in `initialize` anyway.